src/routes/(app)/admin/email-alias/[email]/EmailAliasForm.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
+  import type { SuperValidated } from "sveltekit-superforms";
+  import { superForm } from "sveltekit-superforms/client";
+  import type { EmailAlias, Position } from "@prisma/client";
+  import type { RemovePositionForm, SetCanSendForm } from "./schema";
+
+  export let emailAlias: EmailAlias & { position: Position };
+  export let canSendForm: SuperValidated<SetCanSendForm>;
+  export let isEditing: boolean;
+  const { enhance: canSendEnchance } = superForm(canSendForm, {
+    id: emailAlias.id,
+  });
+
+  let canSendFormHTML: HTMLFormElement;
+
+  export let removePositionForm: SuperValidated<RemovePositionForm>;
+  const { enhance: removePositionEnhance } = superForm(removePositionForm);
+</script>
+
+<div class="flex flex-row items-center justify-between border-b border-neutral">
+  <form
+    action="?/setCanSend"
+    method="post"
+    class="flex flex-row"
+    use:canSendEnchance
+    bind:this={canSendFormHTML}
+  >
+    <div class="my-4 flex flex-row gap-6">
+      <h1 class="font-medium">{emailAlias.position.name}</h1>
+      <input type="hidden" name="aliasId" value={emailAlias.id} />
+      <input type="hidden" name="canSend" value={emailAlias.canSend} />
+      <p class="font-mono">{emailAlias.positionId}</p>
+      <label for="canSend">Kan skicka?</label>
+      {#if isEditing}
+        <input
+          type="checkbox"
+          name="canSend"
+          id="canSend"
+          class="toggle toggle-primary"
+          on:click={async () => {
+            emailAlias.canSend = !emailAlias.canSend;
+            setTimeout(() => {
```

Comment:
This timeout shouldn't be necessasry

Reply:
Tried it but for some reason this doesn't work, don't know why but the `setTimeout` works